content/en/docs/cluster-administration/cluster-settings/log-collections/add-fluentd-as-receiver.md

```diff
@@ -0,0 +1,154 @@
+
+{{</ notice >}}
+
+```yaml
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fluentd-config
+  namespace: default
+data:
+  fluent.conf: |-
+    # Receive logs sending from Fluent Bit on port 24224
+    <source>
+      @type forward
+      port 24224
+    </source>
+
+    # Because this will send logs Fluentd received to stdout,
+    # to avoid Fluent Bit and Fluentd loop logs collection,
+    # add a filter here to avoid sending logs from the default namespace to stdout again
+    <filter **>
+      @type grep
+      <exclude>
+        key $.kubernetes.namespace_name
+        pattern /^default$/
+      </exclude>
+    </filter>
+
+    # Send received logs to stdout for demo/test purpose only
+    # Various output plugins are supported to output logs to S3, MongoDB, Cassandra, MySQL, syslog and Splunk etc.
+    <match **>
+      @type stdout
+    </match>
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: fluentd
+  name: fluentd
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: fluentd
+  template:
+    metadata:
+      labels:
+        app: fluentd
+    spec:
+      containers:
+      - image: fluentd:v1.9.1-1.0
+        imagePullPolicy: IfNotPresent
+        name: fluentd
+        ports:
+        - containerPort: 24224
+          name: forward
+          protocol: TCP
+        - containerPort: 5140
+          name: syslog
+          protocol: TCP
+        volumeMounts:
+        - mountPath: /fluentd/etc
+          name: config
+          readOnly: true
+      volumes:
+      - configMap:
+          defaultMode: 420
+          name: fluentd-config
+        name: config
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: fluentd-svc
+  name: fluentd-svc
+  namespace: default
+spec:
+  ports:
+  - name: forward
+    port: 24224
+    protocol: TCP
+    targetPort: forward
+  selector:
+    app: fluentd
+  sessionAffinity: None
+  type: ClusterIP
+EOF
+```
+
+## Step 2: Add Fluentd as log receiver (aka collector)
+
+1. To add a log receiver:
+
+- Login KubeSphere with an account of ***platform-admin*** role
+- Click ***Platform*** -> ***Clusters Management***
+- Select a cluster if multiple clusters exist
+- Click ***Cluster Settings*** -> ***Log Collections***
+- Log receivers can be added by clicking ***Add Log Collector***
+
+![Add receiver](/images/docs/cluster-administration/cluster-settings/log-collections/add-receiver.png)
+
+2. Choose ***Fluentd*** and fill in the Fluentd service address and port like below:
+
+![Add Fluentd](/images/docs/cluster-administration/cluster-settings/log-collections/add-fluentd.png)
+
+3. Fluentd appears in the receiver list of ***Log Collections*** UI and its status shows ***Collecting***.
+
+![Receiver List](/images/docs/cluster-administration/cluster-settings/log-collections/receiver-list.png)
+
+
+4. Verify Fluentd is receiving logs sent from Fluent Bit:
```

Comment:
Verify Fluentd is receiving logs sent from Fluent Bit:
->
Verify whether Fluentd is receiving logs sent from Fluent Bit: